### PR TITLE
Feature/1.17.0

### PR DESCRIPTION
**Pull request by Sandertv**

### CODE REVIEW

File: minecraft/protocol/item_stack.go [hunk 1]
```diff
@@ -541,6 +543,30 @@ func (c *CraftRecipeOptionalStackRequestAction) Unmarshal(r *Reader) {
...
+	w.Varint32(&c.StackNetworkID)
+}
+
+// Unmarshal ...
+func (c *CraftMineStackRequestAction) Unmarshal(r *Reader) {
+	r.Varint32(&c.Unknown1)
+	r.Varint32(&c.PredictedDurability)
+	r.Varint32(&c.StackNetworkID)
+}
+
```
Comment (Sandertv) [inline minecraft/protocol/item_stack.go:569]: Changes in this file seem to be redundant.

File: minecraft/protocol/packet/start_game.go [hunk 2]
```diff
@@ -190,6 +190,8 @@ type StartGame struct {
 	// is a new system introduced in 1.16. Backwards compatibility with the inventory transactions has to
 	// some extent been preserved, but will eventually be removed.
 	ServerAuthoritativeInventory bool
+	// GameVersion is the version of the game the server is running. The exact function of this field isn't clear.
+	GameVersion string
```
Comment (Sandertv) [inline minecraft/protocol/packet/start_game.go:194]: This exists side by side with the BaseGameVersion?

Comment (JustTalDevelops) [reply minecraft/protocol/packet/start_game.go:194]: Yeah, it's a bit... weird, but that's what it seems to be.

File: minecraft/protocol/item_stack.go (hunk 1 above)
Comment (JustTalDevelops) [reply minecraft/protocol/item_stack.go]: Oops, removed.

**Comment by Sandertv** [comment]
Documentation for new stuff is currently missing, but we'll add that later.

**Pull request merged**